lib/harness/renderer/file.ex

```diff
@@ -110,6 +124,33 @@ defmodule Harness.Renderer.File do
     end
   end
 
+  def generate(%Run{} = run, %__MODULE__{type: :hard_link} = link) do
+    to = Path.join(run.output_directory, link.output_path)
+    from = link.source_path
+
+    # There's no apparent reliable way to determine if an existing file at `to`
+    # is the hard link previously made by harness. Therefore, harness will
+    # always delete the `to` file and recreate the hard link. The `skip_files`
+    # config is available if the user wants to customize the contents of the
+    # `to` file.
+    #
+    # (We could infer that the desired hard link already exists if the contents
```

Comment:
Keep a hard-link manifest somewhere as truth and just ensure links are made from that?